mpas_analysis/polar_regions.cfg

```diff
@@ -80,10 +80,10 @@ diagramType = volumetric
 # if diagramType == 'scatter', only the min and max are important (and the
 #   bins are only used for computing neutral density contours)
 Tbins = numpy.linspace(-2.5, 8.0, 211)
-Sbins = numpy.linspace(33.5, 35.5, 401)
+Sbins = numpy.linspace(30, 35.5, 201)
 
 # density contour interval
-rhoInterval = 0.1
+rhoInterval = 0.2
 
```

Comment:
I'm not sure why we added the `[TSDiagramsForArcticOceanRegions]` section here but I think it makes sense to remove it and instead to make these edits in `default.cfg`.

It looks to me from the plots like `rhoInterval = 0.4` might be more appropriate, given this broad salinity range.

Reply:
Resolved by https://github.com/MPAS-Dev/MPAS-Analysis/pull/896/commits/18fbc747fe8db593cf1b29efa9bc599d546296fd. 

`rhoInterval = 0.3` looked more appropriate than `rhoInterval=0.4`. See https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.cbegeman/analysis/A_WCYCL1850.ne4_oQU480.anvil/clim_3-5_ts_1-5_arcticTSdefault/ocean/index.html

Reply:
Looks good. I think you could even go back to 0.2 with the revised density range you now have.

Reply:
But I'm fine with 0.3.